### ISSUE TITLE
[Bug]: ModuleNotFoundError: No module named 'diffusers'

### ISSUE THREAD

**Issue by satvikreddy** [lshqqytiger/stable-diffusion-webui-directml#192]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

When running webui-user.bat for first time, I am getting the error:

ModuleNotFoundError: No module named 'diffusers'

When I checkout to an old commit via `git checkout 5e48f30`, it works.

Error on current master:
```
venv "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
fatal: No names found, cannot describe anything.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: ## 1.4.1
Commit hash: 99e93f174f783f4848bddeff7cc0d82dacc20719
Installing requirements
Launching Web UI with arguments: --medvram
No module 'xformers'. Proceeding without it.
Traceback (most recent call last):
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\launch.py", line 38, in <module>
    main()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\launch.py", line 34, in main
    start()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\launch_utils.py", line 390, in start
    import webui
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\webui.py", line 42, in <module>
    from modules.call_queue import wrap_gradio_gpu_call, wrap_queued_call, queue_lock  # noqa: F401
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\call_queue.py", line 5, in <module>
    from modules import shared, progress, errors
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\shared.py", line 88, in <module>
    directml_hijack_init()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\__init__.py", line 21, in directml_hijack_init
    import modules.dml.hijack
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\hijack\__init__.py", line 6, in <module>
    import modules.dml.hijack.diffusers
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\hijack\diffusers.py", line 2, in <module>
    import diffusers
ModuleNotFoundError: No module named 'diffusers'
```

### Steps to reproduce the problem

1. Go to master (commit 99e93f1)
2. Install on Windows machine
3. Run webui-user.bat
4. Getting error: ModuleNotFoundError: No module named 'diffusers'

### What should have happened?

It should work in latest master (99e93f1) as it does in the old commit (5e48f30)

### Version or Commit where the problem happens

99e93f1

### What Python version are you running on ?

Python 3.10.x

### What platforms do you use to access the UI ?

Windows

### What device are you running WebUI on?

AMD GPUs (RX 5000 below)

### Cross attention optimization

Automatic

### Command Line Arguments

```Shell
No
```

### List of extensions

No

### Console logs

```Shell
venv "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\venv\Scripts\Python.exe"
fatal: No names found, cannot describe anything.
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Version: ## 1.4.1
Commit hash: 99e93f174f783f4848bddeff7cc0d82dacc20719
Installing requirements
Launching Web UI with arguments: --medvram
No module 'xformers'. Proceeding without it.
Traceback (most recent call last):
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\launch.py", line 38, in <module>
    main()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\launch.py", line 34, in main
    start()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\launch_utils.py", line 390, in start
    import webui
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\webui.py", line 42, in <module>
    from modules.call_queue import wrap_gradio_gpu_call, wrap_queued_call, queue_lock  # noqa: F401
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\call_queue.py", line 5, in <module>
    from modules import shared, progress, errors
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\shared.py", line 88, in <module>
    directml_hijack_init()
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\__init__.py", line 21, in directml_hijack_init
    import modules.dml.hijack
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\hijack\__init__.py", line 6, in <module>
    import modules.dml.hijack.diffusers
  File "D:\One_Constructive\Stable Diffusion\webui\stable-diffusion-webui-directml\modules\dml\hijack\diffusers.py", line 2, in <module>
    import diffusers
ModuleNotFoundError: No module named 'diffusers'
```

**Comment by lshqqytiger** [comment]
Should be fixed in aa02899d39c3f0261db4b4b70b9ce262e61fe440.

**Comment by satvikreddy** [comment]
> Should be fixed in [aa02899](https://github.com/lshqqytiger/stable-diffusion-webui-directml/commit/aa02899d39c3f0261db4b4b70b9ce262e61fe440).

I can confirm that this is working. Appreciate the quick response!
